x/rollup/keeper/msg_server.go

```diff
@@ -227,3 +229,24 @@ func (k *Keeper) SetL1BlockHistory(ctx context.Context, info *derive.L1BlockInfo
 func evmToCosmos(addr common.Address) sdk.AccAddress {
 	return addr.Bytes()
 }
+
+// TODO: This is a temporary change while the rollup module refactor is being done. Change when possible
+func ValidateBasic(m *rollupv1.InitiateWithdrawalRequest) error {
+	// We should ensure that the target field is a valid Ethereum address and that the gas_limit and data field are valid.
+	if common.IsHexAddress(m.Target) {
+		return fmt.Errorf("invalid Ethereum address: %s", m.Target)
+	}
+	// Ensure gas_limit is within a reasonable range
+	// https://insights.deribit.com/industry/ethereums-gas-mechanics/
+	gasLimit := binary.BigEndian.Uint64(m.GasLimit) // size=24 (0x18), offset=64 (0x40)
+	if gasLimit < 21000 || gasLimit > 15000000 {
+		return fmt.Errorf("gas limit must be between 21,000 and 15,000,000: %d", gasLimit)
```

Comment:
Where did this upper bound of `15000000` come from? Are there existing constants or helper funcs from geth that we can reuse here instead of needing to redeclare the gas limit boundaries?